tests/io/v3/vm/networks/test_networks_api.py

```diff
@@ -0,0 +1,284 @@
+'''
+Testing the Networks endpoints
+'''
+import uuid
+
+import pytest
+import responses
+from responses import matchers
+
+NETWORK_BASE_URL = r'https://cloud.tenable.com/api/v3/networks'
+BASE_URL = r'https://cloud.tenable.com'
+NETWORK_ID = str(uuid.uuid1())
+
+
+@responses.activate
+def test_create(api):
+    '''
+    Test case for validating create action of Networks API
+    '''
+    api_resp = {
+        'owner_id': 'e9f23194-adb7-4c02-8632-615c694c787e',
+        'created': '1984-06-02T19:05:00',
+        'modified': '1984-06-02T19:09:00',
+        'scanner_count': 0,
+        'id': 'ca92cffe-8aed-4478-ac76-1acb155bac2a',
+        'name': 'test_network_name',
+        'description': '',
+        'is_default': False,
+        'created_by': 'e9f23194-adb7-4c02-8632-615c694c787e',
+        'modified_by': 'e9f23194-adb7-4c02-8632-615c694c787e',
+        'created_in_seconds': 1639119695,
+        'modified_in_seconds': 1639119695
```

Comment:
Should be ISO8601 for all datetime

Reply:
the field here is `created_in_seconds` should we update those to ISO8601 format as well ?

Reply:
Hmm, that's a good point. I suppose in that case, let's leave as is... since the type is explicit in the name